ui.apps/src/main/content/jcr_root/apps/etoolbox-rollout-manager/clientlibs/rollout-manager-ui/css/console-ui.checkboxlist.css

```diff
@@ -40,67 +52,119 @@
     padding-bottom: 0;
 }
 
-.rollout-manager-nestedcheckboxlist-item .inner-checkbox-option {
-    padding-left:1rem;
-    height: max-content;
-    min-width:9rem;
-}
-
-.rollout-manager-nestedcheckboxlist-item label {
-    margin: 0 .1rem 0 .5rem;
-}
-
 .rollout-manager-new-label {
     color: limegreen;
     font-weight: normal;
 }
 
-.rollout-manager-dialog .coral3-Dialog-wrapper {
-    width: 800px;
-    height: 566px;
+.coral3-Dialog-content .rollout-manager-nestedcheckboxlist-container {
+    overflow-y: auto;
+    height: 206px;
 }
 
-.rollout-manager-dialog .coral3-Checkbox {
-    display: flex;
+.rollout-manager-logger-dialog .coral3-Dialog-wrapper {
+    width: 500px;
+    height: auto;
 }
 
-.rollout-manager-dialog .coral3-Checkbox-description {
-    display: flex;
-    flex-grow: 1;
+#rolloutButton:not([disabled]) {
+    filter: brightness(1.3);
 }
 
-.rollout-manager-dialog coral-checkbox-label {
-    display: flex;
-    flex-grow: 1;
-    justify-content: space-between;
-}
+.rollout-manager-dialog {
+    .coral3-Dialog-wrapper,
+    ._coral-Dialog {
+        width: 800px;
+        height: 566px;
+    }
 
-.rollout-manager-dialog .coral3-Accordion-content {
-    padding-right: 0;
-}
+    .coral3-Checkbox {
+        display: flex;
+    }
 
-.rollout-manager-dialog .coral3-Accordion-header {
-    padding-right: 0;
-}
+    .coral3-Checkbox-description {
+        width: 100%;
+    }
 
-.rollout-manager-nestedcheckboxlist-container {
-    overflow-y: auto;
-    height: 206px;
-}
+    coral-checkbox-label {
+        display: flex;
+        justify-content: space-between;
+        text-transform: none;
+    }
 
-.rollout-manager-logger-dialog .coral3-Dialog-wrapper {
-    width: 500px;
-    height: auto;
-}
+    .coral3-Accordion-content {
+        padding-right: 0;
+    }
 
-.rollout-manager-dialog .rollout-manager-last-rollout-date {
-    z-index: 2;
-}
+    .coral3-Accordion-header {
+        padding-right: 0;
+    }
 
-#rolloutButton:not([disabled]) {
-    filter: brightness(1.3);
+    .rollout-manager-last-rollout-date {
+        z-index: 2;
+    }
 }
 
-/*.coral3-Accordion-label {*/
-/*    font-weight: normal;*/
-/*}*/
+/* Cloud styles */
+.rollout-manager-dialog {
```

Comment:
It seems like super risky for the prod solution.
prefixed _ - means private, so that means the Coral or Granite team can officially, without warning, remove, rename and do whatever they want in any even minor update :)

Reply:
_p.s._ discussed and decided to leave it as is for now, since we didn't find any alternatives